net/HttpRequest.cpp

```diff
@@ -307,7 +307,13 @@ int64_t Request::readData(const char* p, const int64_t len)
     if (_stage == Stage::Header)
     {
         // First line is the status line.
+#if !MOBILEAPP
         if (p == nullptr || len < MinRequestHeaderLen)
+#else
+        // Fix infinite loop on mobile by skipping the minimum request header
```

Comment:
Wouldn't it make more sense to define `MinRequestHeaderLen` differently for mobile, instead of explicitly short-circuiting here?

But even that seems odd. I mean, this is generic HTTP parser. Why would we blanket disable it on Mobile? Doesn't make sense. Surely we use HTTP on Mobile! Perhaps we don't use sockets for certain cases, but disabling such independent code would make it harder to use this code anywhere in Mobile at all.

Reply:
We shouldn't use HTTP on Mobile - we should just push the messages through without wrapping in a websocket container for performance & slickness.